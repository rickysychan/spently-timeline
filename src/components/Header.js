import React from 'react'
import '../css/header.css'
import '../css/colors.css'

const Header = ({ title }) => 
    <div className='header'>
        <h2>{ title }</h2>
        <h5>Jun 1st, 2018 - July 1st, 2018</h5>
        {/* TO DO: Hook up component to date filter  */}
    </div>

export default Header