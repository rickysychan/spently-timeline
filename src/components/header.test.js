import React from 'react'
import { shallow } from 'enzyme'
import Header from './Header'

const header = shallow(<Header />)

describe('Header', () => {
    it('should render properly', () => {
        expect(header).toMatchSnapshot()
    })
})