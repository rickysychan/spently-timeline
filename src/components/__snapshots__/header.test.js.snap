// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render properly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h2 />,
        <h5>
          Jun 1st, 2018 - July 1st, 2018
</h5>,
      ],
      "className": "header",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": undefined,
        },
        "ref": null,
        "rendered": null,
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Jun 1st, 2018 - July 1st, 2018",
        },
        "ref": null,
        "rendered": "Jun 1st, 2018 - July 1st, 2018",
        "type": "h5",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h2 />,
          <h5>
            Jun 1st, 2018 - July 1st, 2018
</h5>,
        ],
        "className": "header",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": undefined,
          },
          "ref": null,
          "rendered": null,
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Jun 1st, 2018 - July 1st, 2018",
          },
          "ref": null,
          "rendered": "Jun 1st, 2018 - July 1st, 2018",
          "type": "h5",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
